config/config.go

```diff
@@ -37,3 +37,5 @@ var (
 	// if RequirePass is set to an empty string, no authentication is required
 	RequirePass string = utils.EmptyStr
 )
+
+var TestEnvEnabled = false
```

Comment:
Let's call this `writeAOFOnCleanup`

Reply:
Updated.
Reasoning behind `TestEnvEnabled` was so that we can reuse this variable for other testing scenarios related configurations in future if required.

Reply:
I feel it is better to have only application specific config flags inside the code, which can be set in a single place using a utility method inside tests. It decouples the activation of different features from each other.